README.md

```diff
@@ -1,54 +1,67 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/mrparkersson-blueviolet)
 
-# Wepback-linters-boilerplate
+# Creating To do List
 
-A starter boilerplate for projects in webpack and linters configs
+> Creating a To Do list where a user adds a to do and he is able to delete the todo
+> Create HTML, CSS file
+> Create JavaScript file which manipulates the DOM; adding clicks to elements and adding fucntions to remove and item from the list
 
 ## Built With
 
 - HTML
 - CSS
 - JavaScript
-- Webpack
+
+## Online Demonstration of my project
+
+[video]
+
+## Online live link
+
+[Visit project online](https://github.com/mrparkersson/leaderboard.git)
 
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+### Setup
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository to your computer using git clone [Git Clone Url](https://github.com/mrparkersson/leaderboard.git)
+- Cd into the folder
+- Open index.html with live server
+- Or you can Fork it instead, and create a pull request to suggest some changes
 
-clone repo: `git clone https://github.com/netman5/webpack-linters-boilerplates.git`
+## Visit And Open Files
 
-then
-`cd webpack-linters-boilerplates`
+[Visit Repo](https://github.com/mrparkersson/leaderboard.git)
 
-### Install
+## Download Repo
 
-run `npm install` to install dependencies
+[Download Repo](https://github.com/mrparkersson/leaderboard.git)
 
 ## Authors
 
-👤 **Ola Ishola**
+👤 **Amo Parker Prince**
 
-- GitHub: [@netman5](https://github.com/netman5)
-- Twitter: [@Orlaish](https://twitter.com/Orlaish)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ola-ishola/)
+- GitHub: [@githubhandle](https://github.com/mrparkersson)
+- Twitter: [@twitterhandle](https://twitter.com/theparkersson)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/amoparkerprince)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/mrparkersson/leaderboard.git).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Inspiration: Microverse
```

Comment:
- [ ] when adding the license please make sure the link is working properly so when you click it you get redirected to the actual licence. please have a look at the [Microverse template](https://github.com/microverseinc/readme-template) to see how this should work.